create_card.py

```diff
@@ -58,7 +58,7 @@ def has_no_user_input(filename, header):
 def create_card(deckpath):
     if not os.path.exists(deckpath):
         call(['mkdir', "-p", deckpath])
-    headers = "{0}{1}{2}{1}".format(draw_frame('QUESTION'), '\n \n \n',
+    headers = "{0}{1}{2}{1}".format(draw_frame('QUESTION'), '\n\n\n',
```

Comment:
I'm not sure why these spaces are here. I assumed it was an error so I corrected it, but it may have been intentional.